app/views/talks/show.html.erb

```diff
@@ -63,7 +63,7 @@
 
   <div class="description">
     <% if contains_html @talk.description %>
-        <%= sanitize @talk.description %>
+        <%= sanitize @talk.description_with_links %>
     <% else %>
         <%= simple_format h(@talk.description) %>
```

Comment:
Unsure if this is ever run with how the current app works. No auto-links in this case.

Reply:
This comment is about the code in the else-clause..

Reply:
Does the description always contain html?
This might be a clause for previous version where we had plain-text

Reply:
I suspect it's from a previous version where we had plain text or both. I'm guessing it doesn't do anything with the current data / how the abstract is created in the app.